src/modules/admin/menu/MenuFormSection.tsx

```diff
@@ -0,0 +1,32 @@
+import { type MenuLayoutVariant } from "@/modules/admin/layout/AdminMenuLayout";
+import { H4 } from "@/modules/common/components/Typography";
+import WireFrame from "@/modules/mock/components/WireFrame";
+import styled from "@emotion/styled";
+import { Card } from "antd";
+
+type MenuFormSectionProps = {
+  menuLayoutVariant: MenuLayoutVariant;
+};
+
+const MenuFormSection: React.FC<MenuFormSectionProps> = () => {
+  return (
+    <MenuFormCard title={<H4>MenuForm</H4>} bordered={false}>
```

Comment:
change "MenuForm" to "Edit"

Reply:
it's probably both edit and add form.

Reply:
Thanks will improve this in sub module
But it can be edit or add